src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package kitchenpos.application;
+
+import kitchenpos.application.fixture.MenuGroupServiceFixture;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@SuppressWarnings("NonAsciiCharacters")
+class MenuGroupServiceTest extends MenuGroupServiceFixture {
+
+    @InjectMocks
+    MenuGroupService menuGroupService;
+
+    @Mock
+    MenuGroupDao menuGroupDao;
+
+    @Test
+    void 메뉴를_등록한다() {
+        // given
+        given(menuGroupDao.save(any(MenuGroup.class))).willReturn(저장된_메뉴_그룹);
+
+        final MenuGroup menuGroup = new MenuGroup(메뉴_그룹_이름);
+
+        // when
+        final MenuGroup actual = menuGroupService.create(menuGroup);
+
+        // then
+        SoftAssertions.assertSoftly(softAssertions -> {
+            softAssertions.assertThat(actual.getId()).isPositive();
+            softAssertions.assertThat(actual.getName()).isEqualTo(menuGroup.getName());
+        });
+    }
+
+    @Test
+    void 메뉴_목록을_조횐한다() {
+        // given
+        given(menuGroupDao.findAll()).willReturn(저장된_메뉴_그룹들);
+
+        // when
+        final List<MenuGroup> actual = menuGroupService.list();
+
+        // then
+        SoftAssertions.assertSoftly(softAssertions -> {
+            softAssertions.assertThat(actual).hasSize(3);
+            softAssertions.assertThat(actual.get(0)).usingRecursiveComparison().isEqualTo(저장된_메뉴_그룹1);
+            softAssertions.assertThat(actual.get(1)).usingRecursiveComparison().isEqualTo(저장된_메뉴_그룹2);
+            softAssertions.assertThat(actual.get(2)).usingRecursiveComparison().isEqualTo(저장된_메뉴_그룹3);
```

Comment:
```suggestion
   softAssertions.assertThat(actual)
                .usingRecursiveComparison()
                .isEqualTo(Arrays.asList(저장된_메뉴_그룹1, 저장된_메뉴_그룹2, 저장된_메뉴_그룹3));
```

이런 방식을 하면 가독성이 조금 더 좋지 않을까요?!

Reply:
개행을 했을 때 가독성이 더 나쁠 것 같아 선택한 방법인데 오리가 말씀해 주신 방법이 더 가독성에 좋네요!
해당 방법으로 수정했습니다.

다만, `usingRecursiveComparison()`의 경우 equals 비교에 있어 필드들을 통해 비교를 한다는 의미기에 테스트와 직접적인 연관은 없다고 생각했습니다.
그래서 해당 메서드는 굳이 개행해주지 않았습니다.